site-src/guides/adapter-rollout.md

```diff
@@ -49,36 +49,7 @@ data:
 
 The new adapter version is applied to the model servers live, without requiring a restart.
 
-
```

Comment:
intro of the file still talks about traffic splitting. 
the original guide was focused on changing the backend model without changing anything in the client request by leveraging traffic splitting (so it was transparent). 

now it’s just add/remove adapter from the pod. 
The guide should probably be updated more than the existing changes.